src/commands/schedule.ts

```diff
@@ -5,13 +5,16 @@ import { extractId } from "./helpers/mentions";
 
 export const schedule: CommandWithInit = {
   init: async (client, services) => {
-    const jobs = await services.store.get<StorableJob[]>("jobs");
-    if (!jobs) {
-      await services.store.set("jobs", []);
-    } else {
-      jobs.forEach((job) => {
-        services.scheduler.scheduleFromStore(job, client);
-      });
+    const guilds = client.guilds.cache.array();
```

Comment:
You can do `.map(async ({ id }) => { /* ... */ })` instead, so we return a `Promise<void>[]` type that we can then do `await Promise.all(jobs)` or something. That way, we are processing the loads concurrently instead of sequentially (so make it a tad faster/more efficient).